Documentation/ApiOverview/Autoloading/Background.rst

```diff
@@ -80,16 +80,14 @@ Using PSR-4 compatible prefix-based resolving
 Instead of looking up every single class and caching the information
 away, composer works on a “prefix”-based resolution. As an example, the
 Composer class loader only needs to know that all PHP classes starting
-with TYPO3\CMS\Core are located within typo3/sysext/core/Classes. The
+with :php:`TYPO3\CMS\Core` are located within :file:`vendor/typo3/cms-core/Classes`
```

Comment:
Please use "EXT:core/Classes" as you want to backport this PR to v11. "vendor/typo3/cms-core" is only true for users with typo3/cms-composer-installers v4 (which is an absolute minority).